src/app/components/media/MediaActionsMenuButton.js

```diff
@@ -127,20 +127,23 @@ class MediaActionsMenuButton extends React.PureComponent {
         onClick={e => this.handleActionAndClose(e, handleItemHistory)}
       >
         <ListItemText
-          primary={<FormattedMessage id="mediaActions.history" defaultMessage="Item history" />}
+          primary={<FormattedMessage id="mediaActions.history" defaultMessage="Item history" description="Menu item to view the history of changes to the item" />}
         />
       </MenuItem>
     ));
 
     return menuItems.length ? (
-      <div>
-        <IconButton
+      <>
+        &nbsp;
+        <ButtonMain
           id="media-actions-menu-button__icon-button"
-          tooltip={<FormattedMessage id="mediaActions.tooltip" defaultMessage="Item actions" />}
```

Comment:
@amoedoamorim I wanted to call your attention to this for some guidance. Before this change, this tooltip attribute was not doing anything, if I looked at the rendered code, it was outputting an object string, so even if it was functioning as a tooltip it wouldn't display correctly:

<img width="203" alt="Screenshot 2023-07-24 at 2 31 57 PM" src="https://github.com/meedan/check-web/assets/418001/cf250a7d-4bc0-436f-8df9-83be04bf8c4f">

Regardless, as you can see i changed it to a title attribute on the Main Button component.

### The question:
Using this format message version it removed the strings from the json file. I just wanted to check if thats ok, or if there is something else I should do? I will note the json files in another comment so you can find them